packages/flutter/lib/src/material/debug.dart

```diff
@@ -24,28 +24,29 @@ bool debugCheckHasMaterial(BuildContext context) {
   assert(() {
     if (context.widget is! Material && context.ancestorWidgetOfExactType(Material) == null) {
       final StringBuffer message = StringBuffer();
-      message.writeln('No Material widget found.');
       message.writeln(
         '${context.widget.runtimeType} widgets require a Material '
-        'widget ancestor.'
+        'widget ancestor, but none was found.\n'
       );
       message.writeln(
+        'Explanation\n'
```

Comment:
since we don't have colour or font size or underlining or anything, maybe make these section headings all-uppercase?